server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/task/TransferTask.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.datatransfer.task;
+
+import java.util.List;
+
+import com.oceanbase.odc.service.datatransfer.model.DataTransferScope;
+import com.oceanbase.tools.loaddump.common.model.ObjectStatus;
+
+public interface TransferTask {
+
+    void init();
+
+    void transfer();
+
+    void destroyQuietly();
+
+    void fail(Throwable throwable);
```

Comment:
`TransferTask` should handle the exception itself, why dose this method exists? who will call it?